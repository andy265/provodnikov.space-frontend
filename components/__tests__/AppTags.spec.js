const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const AppTags = require('~/components/AppTags')

describe('AppTags.vue', () => {
  it('has same HTML structure', () => {
    Object.assign(AppTags, { components: {} })
    let wrapper = shallow(AppTags, {
      propsData: {
        tags: [
          { id: 'tag_id_1', name: 'tag name 1' },
          { id: 'tag_id_2', name: 'tag name 2' }
        ]
      }
    })
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
