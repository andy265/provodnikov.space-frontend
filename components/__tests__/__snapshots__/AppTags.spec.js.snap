// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTags.vue has same HTML structure 1`] = `
<div class="app-tags">
    <AppLink to="/tag/tag_id_1" class="app-tags__item">#tag name 1</AppLink>
    <AppLink to="/tag/tag_id_2" class="app-tags__item">#tag name 2</AppLink>
</div>
`;
